homework1/task05.py

```diff
@@ -0,0 +1,34 @@
+"""
+Given a list of integers numbers "nums".
+
+You need to find a sub-array with length less equal to "k", with maximal sum.
+The written function should return the sum of this sub-array.
+Examples:
+    nums = [1, 3, -1, -3, 5, 3, 6, 7], k = 3
+    result = 16
+"""
+from typing import List
+
+
+def find_maximal_subarray_sum(nums: List[int], k: int) -> int:
+    """Find largest subarray sum with at-least k elements in it.
```

Comment:
Why "at least"? In the task it's said "less [or] equal to k"

Reply:
For example, if we have nums = [1, 3, -1, -3, 5, -3, 6, 7] and k=3, I guess the result should be 13 (6+7)